### PR TITLE
add fiat order history endpoint

### PR DESCRIPTION
**Pull request by 2pd**
- `GET /sapi/v1/fiat/orders` to query user fiat deposit and withdraw history 
    - `GET /sapi/v1/fiat/payments` to query user fiat payments history

### CODE REVIEW

**Review by tantialex — APPROVED** [review]

File: binance/spot/fiat.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    Keyword Args:
+      beginTime (int, optional)
+      endTime (int, optional)
+      page (int, optional): default 1
+      rows (int, optional): default 100, max 500
+      recvWindow (int, optional): The value cannot be greater than 60000
+    """
+
+    payload = {"transactionType": transactionType, **kwargs}
+    return self.sign_request("GET", "/sapi/v1/fiat/orders", payload)
```
Comment (aisling11) [inline binance/spot/fiat.py:19]: missing "check_required_parameter"

File: binance/spot/fiat.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      transactionType (int): 0-buy,1-sell
+    Keyword Args:
+      beginTime (int, optional)
+      endTime (int, optional)
+      page (int, optional): default 1
+      rows (int, optional): default 100, max 500
+      recvWindow (int, optional): The value cannot be greater than 60000
+    """
+
+    payload = {"transactionType": transactionType, **kwargs}
```
Comment (aisling11) [inline binance/spot/fiat.py:39]: missing "check_required_parameter"

File: tests/spot/fiat/test_fiat_payment_history.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    "page": 1,
+    "rows": 100,
+}
+
+
+@mock_http_response(
+    responses.GET,
+    "/sapi/v1/fiat/payments\\?transactionType=0&" + urlencode(params),
+    mock_item,
+    200,
```
Comment (aisling11) [inline tests/spot/fiat/test_fiat_payment_history.py:28]: being perfectionist here, but "," is not needed after 200

File: tests/spot/fiat/test_fiat_order_history.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    "page": 1,
+    "rows": 100,
+}
+
+
+@mock_http_response(
+    responses.GET,
+    "/sapi/v1/fiat/orders\\?transactionType=0&" + urlencode(params),
+    mock_item,
+    200,
```
Comment (aisling11) [inline tests/spot/fiat/test_fiat_order_history.py:28]: Being perfectionist here, but "," is not needed after 200

**Review by aisling11 — APPROVED** [review]

**Pull request merged**